pkg/runtime/scheduler/streamer.go

```diff
@@ -110,23 +109,6 @@ func (s *streamer) handleJob(ctx context.Context, job *schedulerv1pb.WatchJobsRe
 
 	case *schedulerv1pb.JobTargetMetadata_Actor:
 		if err := s.invokeActorReminder(ctx, job); err != nil {
-			// this err is expected if the reminder was triggered once already, the next time it goes to get
-			// triggered by scheduler it sees that it's been canceled and does not invoke the 2nd time. This
-			// more relevant for workflows which currently has a repeat set to 2 since setting it to 1 caused
-			// issues. This will be updated in the future releases, but for now we will see this err. To not
-			// spam users only log if the error is not reminder canceled because this is expected for now.
-			if errors.Is(err, actors.ErrReminderCanceled) {
-				return
-			}
-
-			// If the actor was hosted on another instance, the error will be a gRPC status error,
```

Comment:
This was the change that Elena did this morning. Have you validated that the spam of error logs is not present after removing this?

Reply:
I think we're not going to get this errors anymore, because we're deleting the reminder.

Reply:
I think it's because we're now deleting the reminders when completed and returning nil instead of the original ErrReminderCancelled error

Reply:
We are bringing it back as there was no manual verification that this is not needed anymore. Ideally, we should have automation but, at this stage, we need to be conservative.